flexmeasures/ui/templates/crud/account.html

```diff
@@ -127,7 +130,7 @@ <h3>Assets</h3>
                                 </form>
                                 {% endif %}
                             </th>
-                            <th class="hidden">URL</th>
+                            <th class="hidden d-none invisible" style="display:none;">URL</th>
```

Comment:
Seems like there are 4 ways here to say you want the column hidden.  One should be enough.

Reply:
i have corrected that, just left d-none